index.html

```diff
@@ -128,5 +160,16 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 <!--Font Awesome-->
 <script src="https://kit.fontawesome.com/336a975252.js" crossorigin="anonymous"></script>
+<!-- Navbar hamburger funtion-->
+<script>
+    function myFunction() {
```

Comment:
Rename this function to something meaningful

Reply:
renamed the function.